src/js/polyfills.js

```diff
@@ -0,0 +1,102 @@
+// Production steps of ECMA-262, Edition 6, 22.1.2.1
```

Comment:
Do we need to give credit for these polyfills? If they’re available on npm it might be worth adding them to our build pipeline.

Reply:
Added source links where appropriate.